modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -0,0 +1,195 @@
+
+.Syntax
+[source,terminal]
+----
+$ oc netobserv flows [<feature_command>] [<subcommands>]
+----
+
+.Network Observability Operator feature enablement options
+|===
+|Option| Description| Possible values| Default
+
+| `--enable_pktdrop` 	
+| Enable packet drop.
+| `true`, `false`
+| `false`
+
+| `--enable_rtt`
+| Enable round trip time. 	
+| `true`, `false` 	
+| `false`
+
+| `--enable_dns`
+| Enable DNS tracking. 	
+| `true`, `false` 	
+| `false`
+
+| `--help`
+| Show help.
+| -
+| -
+
+| `--interfaces`
+| Interfaces to match on the flow. For example, "eth0,eth1". 	
+| "<interface>"
+| -
+|===
+
+[id="cli-reference-flow-capture-options"]
+== Flow capture options
+Flow capture has mandatory commands and optional commands. You can use the flow capture options when using the Network Observability Operator feature options.
+
+.`oc netobserv flows` syntax
+[source,terminal]
+----
+$ oc netobserv flows [<feature_command>] [<subcommands>]
+----
+
+.Flow capture filter options
+|===
+|Option| Description| Possible values| Mandatory| Default
+
+| `--enable_filter`
+| Enable flow filter.
+| `true`, `false` 	
+| Yes 
+| `false`
+
+| `--action`
+| Action to apply on the flow.
+| `Accept`, `Reject`
+| Yes
+| `Accept`
+
+| `--cidr`
+| CIDR to match on the flow. 
+| `1.1.1.0/24`, `1::100/64`, or `0.0.0.0/0`
+| Yes
+| `0.0.0.0/0`
+
+| `--protocol`
+| Protocol to match on the flow
+| `TCP`, `UDP`, `SCTP`, `ICMP`, or `ICMPv6` 	
+| No
+| -
+
+| `--direction`
+| Direction to match on the flow
+| `Ingress`, `Egress`
+| No
+| -
+
+| `--dport`
+| Destination port to match on the flow.
+| `80`, `443`, or `49051`
+| no
+| -
+
+| `--sport`
+| Source port to match on the flow.
+| `80`, `443`, or `49051`
+| No
+| -
+
+| `--port`
+| Port to match on the flow.
+| `80`, `443`, or `49051`
+| No 
+| -
+
+| `--sport_range`
+| Source port range to match on the flow.
+| `80-100` or `443-445`
+| No 	
+| -
+
+| `--dport_range` 	
+| Destination port range to match on the flow.
+| `80-100`
+| No
+| -
+
+| `--port_range`
+| Port range to match on the flow.
+| `80-100` or `443-445`
+| No
+| -
+
+| `--icmp_type`
+| ICMP type to match on the flow.
+| `8` or `13`
+| No
+| -
+
+| `--icmp_code`
+| ICMP code to match on the flow.
+| `0` or `1`
+| No
+| - 	
+
+| `--peer_ip`
+| Peer IP to match on the flow.
+| `1.1.1.1` or `1::1`
+| No 	
+| -
+|===
+
+[id="cli-reference-packet-capture-options"]
+== Packet capture options
+You can filter on port and protocol for packet capture data. 
+
+.`oc netobserv packets` syntax
+[source,terminal]
+----
+$ oc netobserv packets [<option>]
+----
+
+.Packet capture filter option
+|===
+|Option| Description| Possible values| Mandatory| Default
+| ` --filter`
+| Enable packet capture filtering.
+| `tcp`, `udp` or `port` You can specify filtering options using a comma as delimeter. For example `--filter=tcp,80` specifies the `tcp` protocol and port `80`.
```

Comment:
Should `port` be `<port>`? Since they seem to have to specify an actual port.

Also some punctuation fixes:
```suggestion
| `tcp`, `udp`, or `<port>`. You can specify filtering options using a comma as delimeter. For example, `--filter=tcp,80` specifies the `tcp` protocol and port `80`.
```
Altho probably could drop the `--filter` part too, since it's assumed, and would make the value fit better in the cell:
```suggestion
| `tcp`, `udp`, or `<port>`. You can specify filtering options using a comma as delimeter. For example, `tcp,80` specifies the `tcp` protocol and port `80`.
```

Reply:
Yeah good call about the port <port> since its a user-specified. And good idea to drop the `--filter`.